templates/index.html

```diff
@@ -449,10 +450,10 @@ <h2 class="u-align--center u-sv3">Cloud Service Providers</h2>
         </div>
         <div class="p-logo-section__item">
           {{ image (
-            url="https://assets.ubuntu.com/v1/093f3200-aws-logo-white-spae.svg",
-            alt="AWS logo",
-            width="145",
-            height="145",
+            url="https://assets.ubuntu.com/v1/4da8004d-amazon-web-services-logo%20@372px.png",
```

Comment:
This is slightly different than the logo we had before, is that expected?

Reply:
Yeah, I think the account manager from us are updating logos from communicating with AWS as one of our business partners. I got it from the [logo directory](https://docs.google.com/spreadsheets/d/1IpUg9W4l54bCw1KoK41RjH9HIrnCh4p5A9lFPrfhtJM/edit#gid=112288557)